spec/features/directly_comment_on_post_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+feature "Comment on a Post" do
+  let(:user) { create(:oliver) }
+  let(:post) { create(:url_post,
+    title: "Y Combinator Is Launching A “Grad School” For Booming Startups") }
+
+  it "allows authenticated user to comment on a post and see new comment on the parent post show view" do
+    submitter = user
```

Comment:
I don't think this is necessary either. Any reason not to just use `user`? Or do `let(:submitted) { create(:oliver) }` on line 4?